### PR TITLE
Aggregate report popup and unresponsive app fixes

### PR DESCRIPTION
**Pull request by yasmin2496**
## Purpose
_Show aggregate report popup and handle adding custody app unresponsiveness_

## Ticket number
_#257_
_#258_

closing #257 
closing #258

### CODE REVIEW

File: src/redux/custodian/reducers/custodian.reducer.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ export default (state = initialState, action) => {
     case ADD_CUSTODY_SUCCESS:
       return {
         ...state,
-        loading: false,
+        loading: true,
```
Comment (RadhikaPPatel) [inline src/redux/custodian/reducers/custodian.reducer.js:226]: loading needs to be updated to false after successful add

Comment (yasmin2496) [reply src/redux/custodian/reducers/custodian.reducer.js:226]: Needs to be there as a workaround to give user feedback that something is still happening in back. Referring to consortium creation for shipment

**Review by RadhikaPPatel — CHANGES_REQUESTED** [review]
Requested changes

**Review by RadhikaPPatel — APPROVED** [review]
Looks good

**Pull request merged**